### PR TITLE
Polysmorps bleed green blood like xenos do, and species without blood don't bleed red

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
BLOOD FOR THE BLOOD GOD

### Intent of your Pull Request
SWEET BLOOD, IT SINGS TO ME

###Explain what the purpose of your PR is. Mention if you tested your changes.
Makes polysmorphs bleed actual xeno blood, since their blood is already established to be acid. It can be scanned and stores the polysmorphs DNA. And yes, i tested it, obviously.

Additionally, species with NOBLOOD species trait, when being hit by projectile, "bleed" grey. Polys have a green effect now.

### Why is this change good for the game?
Makes roundstart race trackable by blood like humans and actual xenos are.

# Wiki Documentation
### Briefly describe your PR and the impacts of it, in layman's terms. 
CRAWLING IN MY SKIN

### What general grouping does this PR fall under? 
BLOOOOD, SO MUCH BLOOD

# Changelog
:cl:  
rscadd: Polysmorphs now bleed green like they always should have.
bugfix: species with no blood no longer bleed red when hit by ballistic projectiles
imageadd: added generic blood splatter
/:cl:

### CODE REVIEW

**Review by nesisknesis — APPROVED** [review]

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Refactor it to be species based maybe. This is a very hacky way to do it, no need to hardcode it like this.
Don't see why you're making NOBLOOD species bleed either, just fix the bug??

**Comment by ChesterTheCheesy** [comment]
> 
> 
> Refactor it to be species based maybe. This is a very hacky way to do it, no need to hardcode it like this.
> Don't see why you're making NOBLOOD species bleed either, just fix the bug??

See [#11066], and i'm making NOBLOOD species have gray particles when hit by projectile, instead of red (which is what is current).

**Comment by Bibby0110** [comment]
> > Refactor it to be species based maybe. This is a very hacky way to do it, no need to hardcode it like this.
> > Don't see why you're making NOBLOOD species bleed either, just fix the bug??
> 
> See [#11066], and i'm making NOBLOOD species have gray particles when hit by projectile, instead of red (which is what is current).

What he meant is that you should do it on the actual species.

The species subtypes of humans are just humans with that species applied when it spawns.

You need to do this on the actual datum species.

Didn't realize the projectile code was for the temporary splatter, my bad, that part looks good.

**Comment by ChesterTheCheesy** [comment]
> You need to do this on the actual datum species.

Yeah? Tell me how you're gonna change a bleed effect on a datum.

**Comment by Bibby0110** [comment]
> > You need to do this on the actual datum species.
> 
> Yeah? Tell me how you're gonna change a bleed effect on a datum.

Same way everything is done on species? Delegate it to a spec_whatever() if the carbon has dna.species

**Comment by MegaEmpirical** [comment]
...why is a plasmaman going to bleed grey now?

**Comment by ChesterTheCheesy** [comment]
> 
> 
> ...why is a plasmaman going to bleed grey now?

They don't bleed, it's the effect that happens when you're hit by a ballistic projectile, like a bullet. Currently all humanoid mobs "bleed red" that way, this only makes that effect grey.

**Comment by MegaEmpirical** [comment]
.. Have I seriously never fucking noticed this? Going to check tomorrow when I get on, that would be so dumb. They shouldn't bleed at all
